api/v1alpha1/freight_helpers.go

```diff
@@ -129,6 +129,19 @@ func (f *Freight) IsVerifiedIn(stage string) bool {
 	return verified
 }
 
+// IsVerifiedInAll returns whether the Freight has been verified in all
+// Stages.
+func (f *Freight) IsVerifiedInAll(stages []string) bool {
+	// NB: This method exists for convenience. It doesn't require the caller to
+	// know anything about the Freight status' internal data structure.
```

Comment:
Nit: We could lose this comment. I know `IsVerifiedIn()` has a comment like this, but it was there to justify its existence when `freight.Status.VerifiedIn[stage]` seems no harder than `freight.IsVerifiedIn(stage)`. `IsVerifiedInAll()` is doing a more obvious bit of work, so I don't think it needs any justification.